typescript-sdk/integrations/pydantic-ai/examples/README.md

```diff
@@ -0,0 +1,60 @@
+# Pydantic AI AG-UI Examples
+
+This directory contains example usage of the AG-UI adapter for Pydantic AI. It provides a FastAPI application that demonstrates how to use the Pydantic AI agent with the AG-UI protocol.
+
+## Features
+
+The examples include implementations for each of the AG-UI dojo features:
+- Agentic Chat
+- Human in the Loop
+- Agentic Generative UI
+- Tool Based Generative UI
+- Shared State
+- Predictive State Updates
+
+## Setup
+
+### Using uv (Recommended)
+
+1. Install dependencies:
+   ```bash
+   uv sync
+   ```
+
+2. Run the development server:
+   ```bash
+   uv run dev
+   ```
+
+### Using pip
+
+1. Install dependencies:
+   ```bash
+   pip install -r requirements.txt
```

Comment:
There's actually no `requirements.txt`, I think we should just always recommend using `uv`

Reply:
Oh shoot yeah that's a holdover from before I figured out the uv dynamic versions thing.